symplyphysics/laws/waves/displacement_in_standing_wave.py

```diff
@@ -0,0 +1,71 @@
+from sympy import Eq, sin, cos, symbols, Function as SymFunction
+from symplyphysics import (
+    units,
+    angle_type,
+    Symbol,
+    Quantity,
+    print_expression,
+    validate_input,
+)
+from symplyphysics.core.quantity_decorator import validate_output_same
+
+# Description
+## A standing, or stationary, wave is the result of the interference of two identical waves
+## moving in the opposite direction. For a wave occuring along a string, the standing wave 
+## is described by the following equation:
+
+# Law: u(x, t) = 2 * u_max * sin(k * x) * cos(w * t)
+## u(x, t) - displacement from rest (position, pressure, electric field, etc)
+## u_max - wave amplitude
+## k - angular wavenumber
+## x - spatial variable
+## w - angular frequency
+## t - time
+
+# Note
+## - The form of the traveling waves the standing wave is the sum of is `u_max * sin(k * x + w * t)`
```

Comment:
Does not look like a proper text. Please rewrite.

Reply:
Okay